lib/h3/traversal.rb

```diff
@@ -32,6 +32,23 @@ module Traversal
     # @return [Integer] Distance between indexes.
     attach_function :h3_distance, :h3Distance, %i[h3_index h3_index], :k_distance
 
+    # @!method h3_line_size(origin, destination)
+    #
+    # Derive the number of hexagons present in the distance between two H3 indexes.
```

Comment:
On the other hand, I think the user may be wondering how is this different from `h3_distance`, when should pick one or the other, etc., since "distance" in this library is also a concept related to "lines".

What do you think about saying that the line size [is simply](https://github.com/uber/h3/blob/b314e6dec5b45ca07868f48564a7ec2479746beb/src/h3lib/lib/localij.c#L559-L562) `h3_distance(origin, destination) + 1` when the distance is computable?

Reply:
Yes that makes sense! The H3 doc says the same thing: https://github.com/uber/h3/commit/7fb02054a4181ce176d632a0d5ff95842a01fc2d